sandbox/grommet-app/src/pages/layouts/kinds/RecordDetail/RecordDetail.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Box,
+  Button,
+  Grid,
+  Notification,
+  Page,
+  PageContent,
+  PageHeader,
+} from 'grommet';
+import { RoutedAnchor } from '../../../../components';
+import { Close, Previous } from 'grommet-icons';
+import ContentPane from '../../../../components/ContentPane';
+import { DetailSummary } from './DetailSummary';
+import { ScheduledActions } from './ScheduledActions';
+import { ILOSecurity } from './ILOSecurity';
+import { RecentActivity } from './RecentActivity';
+import { ScheduledJobs } from './ScheduledJobs';
+
+export const RecordDetail: React.FC = () => {
+  const [scheduledJobs, setScheduledJobs] = React.useState<boolean>(false);
+  const columns = scheduledJobs ? ['flex', 'auto'] : ['flex', 'medium'];
+  const rows = ['auto'];
+
+  const areas = [
+    ['notification', 'context-pane'],
+    ['details', 'context-pane'],
+    ['scheduled-actions', 'context-pane'],
+    ['iLO-security', 'context-pane'],
+    ['unassigned', 'context-pane'],
+  ];
+
+  return (
+    <Page pad={{ bottom: 'xlarge' }} flex="grow">
+      <PageContent>
+        <PageHeader
+          title="Server group name"
+          subtitle="2 servers"
+          parent={<RoutedAnchor as={Link} to="/layouts" label="Layouts" icon={<Previous />} />}
+        />
+        <Grid areas={areas} columns={columns} rows={rows} gap="large">
+          <Box gridArea="notification">
+            <Notification
+              status="info"
+              message="1 job is scheduled."
+              actions={[{
+                label: 'View',
+                href: '#',
+                onClick: () => { setScheduledJobs(true) },
+              }]}
+            />
+          </Box>
+          <ContentPane
+            gridArea="details"
+            heading="Details"
+            level={2}
+            actions={undefined}
+            skeleton={undefined}
+          >
+            <DetailSummary />
+          </ContentPane>
+          <ContentPane
+            gridArea="scheduled-actions"
+            heading="Scheduled actions"
```

Comment:
<img width="1202" alt="Screenshot 2025-01-27 at 9 40 48 PM" src="https://github.com/user-attachments/assets/f7a7b62b-0679-47f4-8a8e-9c367171df3b" />

Getting some overlap here -- not sure if that content pane should have overflow or the page should respond earlier to a single column layout

Reply:
Made adjustments to improve responsive behavior.